packages/safety-suite-api/src/api/dashboardSink/accidents.ts

```diff
@@ -0,0 +1,51 @@
+import {Request} from '@idelic/safety-net';
+import {ApiOptions, ApiSuccessResponse, runApi} from 'src';
```

Comment:
Absolute imports won't work when built. For some reason it doesn't get compiled properly and treats them like a package. Also don't import from the root `index.ts` since it will create a dependency cycle.